myhpom/migrations/0010_state_requirements.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('myhpom', '0009_advancedirective'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='StateRequirement',
```

Comment:
This migration appears to need a refresh - when I try this locally I need to run makemigrations again to capture the help_text changes.

Reply:
Ah, ok, sorry about that.

Reply:
And also the addition of a validator to the AdvanceDirective model